principles.md

```diff
@@ -18,8 +18,8 @@ Outer layers should depend on inner layers (at the source-level), and not vice v
 
 For example:
 * Most services we build are simple forms
-* We should avoid Business Logic in the User Interface. If the project requires it, we **should** include an Architecture Decision Record explaining why.
-* Avoid using JavaScript in the User Interface in the project. The project should **should** include an Architecture Decision Record explaining why. Even if the project uses standard GOV.uk/local authorities' approved components.
+* We should avoid Business Logic in the User Interface. If the project requires it, we **should** include an [Architecture Decision Record](./principles/communicate_into_the_future.md) explaining why.
+* Avoid using JavaScript in the User Interface in the project. The project should **should** include an [Architecture Decision Record](./principles/communicate_into_the_future.md) explaining why. Even if the project uses standard GOV.uk/local authorities' approved components.
```

Comment:
Consistently end bullets with no period (as the first bullet in the list). And remove double `should`.

```suggestion
* Avoid using JavaScript in the User Interface in the project. The project **should** include an [Architecture Decision Record](./principles/communicate_into_the_future.md) explaining why. Even if the project uses standard GOV.uk/local authorities' approved components
```